install_config/syncing_groups_with_ldap.adoc

```diff
@@ -860,3 +867,277 @@ users: <5>
 <4> The name of the Group as specified by the sync file.
 <5> The users that are members of the Group, named as specified by the sync file.
 ====
+
+[[recursive-sync]]
+== Recursive Membership Sync Example
+
+Groups in OpenShift do not nest.  Thus the LDAP server must flatten group membership before
+the data can be consumed.  Microsoft's Active Directory Server supports this feature via the
+https://msdn.microsoft.com/en-us/library/aa746475(v=vs.85).aspx[LDAP_MATCHING_RULE_IN_CHAIN]
```

Comment:
Looks fine to me:

![image](https://cloud.githubusercontent.com/assets/7001223/25751074/cf726210-3181-11e7-85f3-6ea500746af3.png)